### DISCUSSION TITLE
2022-23: 新轮子 globiter 和一些想法

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#358] [Announcements]
# 2022-23: 新轮子 globiter 和一些想法

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-23/

**Comment by sundy-li** [comment]
opendal 可以划出去搞个创业项目了！

**Comment by ImSingee** [comment]
```
COPY BETWEEN dropbox://path/to/dir onedrive://path/to/dir；
```

rclone: 喵喵喵？
